src/smart-components/user/user.scss

```diff
@@ -1,3 +1,9 @@
 .ins-c-rbac__user-label-skeleton {
   width: 50px;
 }
+
+// Chrome-specific fix for cropped  placeholder: https://github.com/patternfly/patternfly/issues/3449
+_:lang(x)::-internal-media-controls-overlay-cast-button, 
+input#filter-by-username.ins-c-conditional-filter[type="text"] {
```

Comment:
Good catch! I'd prefer here setting the `min-width` value for all browsers. It would look a bit off if user openes it in one browser and see the filter a bit smaller. Also any reason why to use `!important`? If we can avoid it it's always better to use more complex selectos

Reply:
@karelhala It worked without !important - done.